### DISCUSSION TITLE
Movable and resizable ContentDialog when in overlay mode.

### DISCUSSION THREAD

**Discussion by saibot75** [amwx/FluentAvalonia#268] [Ideas]
In overlay mode, would it be possible to implement ability to move and resize the overlayed ContentDialog? In WASM we are forced to use overlay since there is no support for multiple windows and loosing the ability to move and resize is a little bummer.

**Comment by amwx** [comment]
It would be possible, yes, but with a bit of work. At least right now, I'm not sure I'd want this as a standard feature on the dialogs.

To do it, you'd just have to get ahold of the root of the actual dialog from the template - keeping in mind in overlay mode there is that "smokescreen" that darkens the rest of the app (which is just a panel with a semi-transparent background). You should then be able to just hook up some pointer events and adjust the Margins to move it around.
